server/seed.py

```diff
@@ -0,0 +1,77 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from app import create_app
+from datetime import datetime
+from app.models import User, Invitations, Store, Product, Inventory, Request
+from dotenv import load_dotenv
+from app import db
+
+
+
+# Initialize Flask app
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////instance/app.db' 
```

Comment:
Make sure you have provided the correct URL for db seeding. I can see this is pointing to an SQLite database, but should be pointing to something on postgres.

Reply:
I have updated the seed file and was able to successfully seed my postgres db